app/src/test/java/com/pelukron/ListaTest.java

```diff
@@ -0,0 +1,30 @@
+package com.pelukron;
+
+import jdk.jfr.Description;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ListaTest {
+
+    Lista list;
+    @BeforeEach void beforeEach(){
+        list = new Lista();
+        list.setPrimerNodo(new Nodo(1L));
+    }
+    
+    /**
+     * Validate when there aren't more node linked
+     */
+    @Test void eliminarPrimerNodo1() {
```

Comment:
Annotations are usually placed above the test

Reply:
Sorry, don't get this comment

Reply:
Like this:

` @BeforeEach 
   void beforeEach(){
        list = new Lista();
        list.setPrimerNodo(new Nodo(1L));
    }

  @Test 
   void eliminarPrimerNodo1() { ...`

Reply:
I saw new decorator order is like that but i would like to set a linter as node project to avoid this kind of "issues"